Backend.dockerfile

```diff
@@ -18,7 +18,7 @@ COPY pkg-svcs/backend/ /app/backend/
 
 FROM source as builder
 WORKDIR /app/backend
-RUN mv .env.registry .env
```

Comment:
@heidermassu I do not think we need this change. We will override any env var from the k8s service manifest

Reply:
Line removed. That worked.